setup.py

```diff
@@ -15,40 +15,35 @@
     README = readme.read()
 
 with open('test_requirements.txt') as test_reqs:
-    tests_require = test_reqs.readlines(),
+    TESTS_REQUIRE = test_reqs.readlines(),
 
 
-class PyTest(test):
-    """
-    Test runner for package.
-    """
-    user_options = test.user_options[:]
+class PyTest(testcommand):
+    """PyTest class to enable running `python setup.py test`."""
+    user_options = testcommand.user_options[:]
     user_options += [
         ('coverage', 'C', 'Produce a coverage report for orcoursetrion'),
         ('pep8', 'P', 'Produce a pep8 report for orcoursetrion'),
-        ('flakes', 'F', 'Produce a flakes report for orcoursetrion'),
-
+        ('pylint', 'l', 'Produce a pylint report for orcoursetrion'),
     ]
     coverage = None
     pep8 = None
-    flakes = None
+    lint = None
     test_suite = False
     test_args = []
 
     def initialize_options(self):
-        test.initialize_options(self)
+        pass
```

Comment:
Based on https://pytest.org/latest/goodpractises.html#integrating-with-distutils-python-setup-py-test and I didn't look too closely, but this is wrong regardless since it is calling itself and not super.